core/chains/evm/logpoller/parser.go

```diff
@@ -272,15 +272,14 @@ func (v *pgDSLParser) whereClause(expressions []query.Expression, limiter query.
 			return "", errors.New("invalid cursor direction")
 		}
 
-		block, logIdx, txHash, err := valuesFromCursor(limiter.Limit.Cursor)
+		block, logIdx, _, err := valuesFromCursor(limiter.Limit.Cursor)
```

Comment:
I have a feeling most products are going to use _ here to ignore txhash also.  I still can't think of any potential use a product would have for getting a txhash back here... it just doesn't seem relevant to the feature they are using.

I think the PR as it stands now will give correct results, so I'm willing to approve it if everyone else is happy.  But just noting that I still think it would make a lot more sense to not involve txhash here... it's harmless, but doesn't add any value either IMO